### PR TITLE
finished new edit-title button

### PR DESCRIPTION
**Pull request by Dtphelan1**
Modified the edit-note-button stylign to reflect the mockups. This not only included modifying the actual icon to match the mockups, but also modifying the alignment s.t. the icon-button is in the left gutter of the note panel, visually separating it from the text of the note description. 

_Pull requests into Flux require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:._

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome
- [x] Manually tested in IE

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- **N/A** Cheat sheet is updated
- **N/A** Demo script is updated 
- **N/A** Documentation on Wiki has been updated 
- **N/A** Additional technical components and libraries are documented and added to wiki as needed

**NoteParser**

- **N/A** Note parser has been updated if structured phrases change

**Code Quality**

- [x] 4-space indents - convert any tabs to spaces
- [x] Use public class field syntax for binding functions - ex. handleChange = () => { ... };
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- **N/A** Added Enzyme-UI tests for slim mode 
- **N/A** Added Enzyme-UI tests for full mode
- **N/A** Added backend tests for new functionality added
- **N/A** Added note parser examples to test new functionality

## Reviewer 1: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

## Reviewer 2: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: src/notes/FluxNotesEditor.scss [hunk 1]
```diff
@@ -24,12 +26,17 @@
...
+            font-size: 0.8rem;
+            margin: 0px;
+            padding-left: 0.5rem;
             font-weight: bold;
             
-            #edit-note-name-btn:hover {
-                cursor: pointer;
+            #edit-note-name-btn {
+                margin-left: -23px;
+                margin-right: 3px;
```
Comment (mgramigna) [inline src/notes/FluxNotesEditor.scss]: [MG1] Very small thing, but I would consider playing around with the margin-right value a little more. I think the text looks a bit more aligned with "Signed By" and "Source" with a value of 7px or 6px. Right now the note name is slightly to the left of the rest of the text.

File: src/notes/FluxNotesEditor.jsx [hunk 2]
```diff
@@ -1342,6 +1348,26 @@ class FluxNotesEditor extends React.Component {
             );
         }
     }
+
+    EditNoteTitleButton = () => {
```
Comment (mgramigna) [inline src/notes/FluxNotesEditor.jsx]: [MG2] Code style: I would change EditNoteTitleButton to be camelcase (lowercase e) and add a semicolon after return.

**Comment by gregquinn2001** [comment]
Watch out for rebase between this one and Nicole's pull request!

**Comment by gregquinn2001** [comment]
[GQ1] What do you think about this? I feel like the icon doesn't line up with the title text. Perhaps it's the horizontal line in the icon not being at the baseline of the text...

**Comment by gregquinn2001** [comment]
[GQ2] Say no on this one or create a JIRA if you want, but do you think clicking on the icon or the title itself should go into edit mode for the title?

**Comment by Dtphelan1** [comment]
Fixed GQ2 I think, GQ1 I can play with tomorrow morning! :)

**Comment by Dtphelan1** [comment]
I've made minor improvments to the pencil icon positioning that might serve to solve GQ1. I think the hard part is that the eye want the pencil-tip, not the underline, to share the baseline with the text. I've changed styling so that the underline is matched with the baseline (which I wasn't even doing 100% before) but it might still look a little off. Let me know what you think.

**Comment by Dtphelan1** [comment]
Update: I went with a negative bottom margin, as it looks a little more like the mockups. Let me know what you all think.

**Comment by Dtphelan1** [comment]
Greg stop working at 8 PM on Fridays!   
  

  
  

  
  
>

**Pull request merged**